site/app/libraries/database/DatabaseQueries.php

```diff
@@ -6423,6 +6428,9 @@ public function changeQueueRegex($regex_pattern, $queue_code) {
         $this->course_db->query("UPDATE queue_settings SET regex_pattern = ? WHERE code = ?", [$regex_pattern, $queue_code]);
     }
 
+    public function changeQueueContactInformation($contact_information, $queue_code) {
```

Comment:
Would recommend adding PHP type hints to newly added functions whenever possible 

https://submitty.org/developer/coding_style_guide/php#type-declarations